projects/project.json

```diff
@@ -0,0 +1,17 @@
+{
+    "projects": [
+        {
+            "name": "falco",
+            "organization": "falcosecurity",
+            "sub_components": [
+                "ebpf",
+                "modern-ebpf",
+                "kmod"
+            ]
+        },
+        {
+            "name": "kepler",
```

Comment:
Kepler should not be here. Falco is the only project we are measuring.

Reply:
yeah will remove it 😄